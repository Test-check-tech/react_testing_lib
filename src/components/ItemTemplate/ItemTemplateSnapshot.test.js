import React from 'react';
import { render } from '@testing-library/react';
import Template from './ItemTemplate';

it("Snapshot", () => {
  const { asFragment,getByText }   = render(<Template />);
  expect(asFragment()).toMatchSnapshot();  

});