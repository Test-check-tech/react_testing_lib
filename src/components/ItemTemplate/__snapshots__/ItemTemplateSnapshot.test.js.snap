// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot 1`] = `
<DocumentFragment>
  <div>
    <div>
      <h1
        align="center"
      >
        Item Creation
      </h1>
      <hr
        class="new"
      />
    </div>
       
    <b>
      Task Type :
    </b>
     
    <select
      id="dropdown"
    >
      <option>
        Select Task Type
      </option>
      <option
        value="1"
      >
        Multiple Choice - Keyed
      </option>
      <option
        value="2"
      >
        True/False
      </option>
      <option
        value="3"
      >
        Multiple Choice - Scaled
      </option>
      <option
        value="4"
      >
        Multiple Choice - Multiple Selection
      </option>
      <option
        value="5"
      >
        Multi-Part
      </option>
    </select>
          
    <b>
      Task Sub-Type :
    </b>
     
    <select
      disabled=""
      id="dropdown"
    >
      <option>
        Select Task Sub-Type
      </option>
      <option
        value=""
      />
      <option
        value=""
      />
      <option
        value=""
      />
      <option
        value=""
      />
      <option
        value=""
      />
    </select>
    <hr />
    <style>
      .tab {position:absolute;left:750px; } 
    </style>
    <style>
      .new {border: 2px solid green; } 
    </style>
  </div>
</DocumentFragment>
`;
